client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -425,14 +431,17 @@ const mapStateToProps = ( state, { postId, postType, duplicatePostId }: Props )
 
 	const iframeUrl = addQueryArgs( queryArgs, siteAdminUrl );
 
+	// Prevents the iframe from loading using a cached frame nonce.
```

Comment:
Thanks for that comment

Reply:
I think @mmtr gets credit for that from a previous version :)